src/pages/currency-trading/forms/TransactionForm.js

```diff
@@ -287,12 +275,45 @@ export default function TransactionForm({ recordId, labels, maxAccess, plantId,
       issue_country: yup.string().required(),
       nationality: yup.string().required(),
       cell_phone: yup.string().required(),
-      profession: yup.string().required()
+      profession: yup.string().required(),
+      rows: yup
```

Comment:
Good stuff but please don't use rows or rows2 as names.
Use operations and amount. rows can be very confusing.